<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EmployeeEditor</class>
 <widget class="QWidget" name="EmployeeEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>602</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="cmdAddNew">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Add New</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/images/text-plus-icon.png</normaloff>:/images/text-plus-icon.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cmdDelete">
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/images/Delete-icon.png</normaloff>:/images/Delete-icon.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Basic Information</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="6" column="2">
        <widget class="QLineEdit" name="txtNHIFNo"/>
       </item>
       <item row="5" column="2">
        <widget class="QLineEdit" name="txtPINNo"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>&amp;ID No.:</string>
         </property>
         <property name="buddy">
          <cstring>txtIDNo</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QLineEdit" name="txtIDNo"/>
       </item>
       <item row="6" column="0" colspan="2">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>NH&amp;IF No:</string>
         </property>
         <property name="buddy">
          <cstring>txtNHIFNo</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>&amp;Last Name:</string>
         </property>
         <property name="buddy">
          <cstring>txtLastName</cstring>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Pin &amp;No:</string>
         </property>
         <property name="buddy">
          <cstring>txtPINNo</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>&amp;Middle Name:</string>
         </property>
         <property name="buddy">
          <cstring>txtMiddleName</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLineEdit" name="txtMiddleName"/>
       </item>
       <item row="3" column="2">
        <widget class="QLineEdit" name="txtLastName"/>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>&amp;First Name:</string>
         </property>
         <property name="buddy">
          <cstring>txtFirstName</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLineEdit" name="txtFirstName"/>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Address:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="3">
        <widget class="KMessageWidget" name="basicDetailsMessage" native="true">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>10</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="13" column="2">
        <widget class="QComboBox" name="cboDepartment"/>
       </item>
       <item row="14" column="2">
        <widget class="QComboBox" name="cboJobGroup"/>
       </item>
       <item row="15" column="3">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="13" column="0" colspan="2">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>&amp;Department:</string>
         </property>
         <property name="buddy">
          <cstring>cboDepartment</cstring>
         </property>
        </widget>
       </item>
       <item row="9" column="2">
        <widget class="QLineEdit" name="txtEmail"/>
       </item>
       <item row="7" column="0" colspan="2">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>&amp;NSSF No:</string>
         </property>
         <property name="buddy">
          <cstring>txtNSSFNo</cstring>
         </property>
        </widget>
       </item>
       <item row="8" column="2">
        <widget class="QLineEdit" name="txtTel"/>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>&amp;Email:</string>
         </property>
         <property name="buddy">
          <cstring>txtEmail</cstring>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>&amp;Tel:</string>
         </property>
         <property name="buddy">
          <cstring>txtTel</cstring>
         </property>
        </widget>
       </item>
       <item row="14" column="0" colspan="2">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>&amp;Job Group:</string>
         </property>
         <property name="buddy">
          <cstring>cboJobGroup</cstring>
         </property>
        </widget>
       </item>
       <item row="7" column="2">
        <widget class="QLineEdit" name="txtNSSFNo"/>
       </item>
       <item row="11" column="2">
        <widget class="QLineEdit" name="txtPostcode"/>
       </item>
       <item row="10" column="2">
        <widget class="QLineEdit" name="txtAddress"/>
       </item>
       <item row="11" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Postcode:</string>
         </property>
        </widget>
       </item>
       <item row="12" column="0">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Town:</string>
         </property>
        </widget>
       </item>
       <item row="12" column="2">
        <widget class="QLineEdit" name="txtTown"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Payment Details</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>New Payment</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="text">
            <string>Delete Selected</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="tableView"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KMessageWidget</class>
   <extends>QWidget</extends>
   <header location="global">kmessagewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cmdAddNew</tabstop>
  <tabstop>cmdDelete</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>txtFirstName</tabstop>
  <tabstop>txtMiddleName</tabstop>
  <tabstop>txtLastName</tabstop>
  <tabstop>txtIDNo</tabstop>
  <tabstop>txtPINNo</tabstop>
  <tabstop>txtNHIFNo</tabstop>
  <tabstop>txtNSSFNo</tabstop>
  <tabstop>txtTel</tabstop>
  <tabstop>txtEmail</tabstop>
  <tabstop>txtAddress</tabstop>
  <tabstop>txtPostcode</tabstop>
  <tabstop>txtTown</tabstop>
  <tabstop>cboDepartment</tabstop>
  <tabstop>cboJobGroup</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>tableView</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
